src/osip-monitoring-view/src/main/java/edu/kit/pse/osip/monitoring/view/settings/SettingsMainWindow.java

```diff
@@ -120,6 +120,11 @@ private void createLayout() {
         Scene scene = new Scene(generalLayout);
         window.setTitle(translator.getString("monitoring.settings.title"));
         window.getIcons().add(new Image("icon.png"));
+        window.setOnCloseRequest(event -> {
+            if (buttonCancel.isDisabled()) {
+                event.consume();
+            } 
+        });
```

Comment:
Für das Beenden der Überwachungskonsole ist der x Button des Hauptfensters da. Ich schließe mich hier David an: entweder der Nutzer gibt korrekte Daten ein oder er kann das Einstellungsfenster nicht schließen, oder (als letzte Möglichkeit) das x des Einstellungsfensters schließt in einem solchen Fall ebenfalls die komplette Überwachungskonsole. Beides widerspricht den üblichen Erwartungen. Aber ansonsten kann der Nutzer das Einstellungsfenster einfach schließen und dann? Dann kann er mit der Überwachungskonsole nichts anfangen.